code/game/objects/structures/tables_racks.dm

```diff
@@ -124,7 +124,7 @@
 		if(!G.confirm())
 			return 0
 		G.affecting.loc = src.loc
-		G.affecting.Weaken(5)
+		G.affecting.Weaken(2)

```

Comment:
Are we haggling? I can go as low as 4. I gotta make a profit, ok, I'm running a business here.